plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -93,6 +93,8 @@ func init() {
 	factory.RegisterFitPredicate("HostName", predicates.PodFitsHost)
 	// Fit is determined by node selector query.
 	factory.RegisterFitPredicate("MatchNodeSelector", predicates.PodSelectorMatches)
+	// Optional, cluster-autoscaler friendly priority function - give used nodes higher priority.
+	factory.RegisterPriorityFunction("MostRequestedPriority", priorities.MostRequestedPriority, 1)
```

Comment:
> Why? I looked into the code and it seems that when you define the policy-config, then it doesn't use any default predicates/priorities - it only uses the ones defined in you policy config.

Oh sorry, you're right.

And the idea about using --algorithm-provider sounds good -- I had forgotten about that.